Source/HOHQMeshMain.f90

```diff
@@ -162,13 +161,13 @@ SUBROUTINE ReadCommandLineArguments(version, test, generateTest, controlFileName
 
 
          IF ( CommandLineArgumentIsPresent("-version") )     THEN
-            PRINT *, "HOMesh Version ", version
+            PRINT *, "HOHQMesh Version ", version
             STOP
          END IF
 
          IF ( CommandLineArgumentIsPresent("-help") )     THEN
```

Comment:
```suggestion
         IF ( CommandLineArgumentIsPresent("-help") .OR. CommandLineArgumentIsPresent("--help") )     THEN
```
I know that the HOHQMesh standard is to eschew command line arguments with two dashes, but OTOH, `--help` is _the_ go-to command I usually run first for an unknown program, thus maybe it wouldn't hurt to allow it just for the help?

Reply:
Actually, I wouldn't modify it 😅: I'd say `-help` is the correct command to use, and `--help` is just added for sanity for users who do not bother to read the docs but like to explore things interactively (such as myself 😬).

But I have no strong opinion on this, thus I'm good either way.

Reply:
Okay, I don't really mind either way. I will resolve my conversation below.